### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by rachelhiayouappi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,8 @@ Load the project with Android Studio and Gradle will do the rest.
 
 * On the project run menu select the desired demo app you would like to run.
 * Press the run button and wait for the demo app to load.
+
+
+### How do I get the latest SDK? ###
+* Please make sure to take the assets from the support website.
```
Comment (guybashan) [inline README.md:22]: You can download our latest SDK from the support web site:
- SDK: ....
- AdMob: ....
- MoPub: ....
- Native: ....

File: README.md [hunk 2]
```diff
@@ -16,3 +16,8 @@ Load the project with Android Studio and Gradle will do the rest.
 
 * On the project run menu select the desired demo app you would like to run.
 * Press the run button and wait for the demo app to load.
+
+
+### How do I get the latest SDK? ###
+* Please make sure to take the assets from the support website. 
+* Note that the demo apps listed here do not always contain the updated SDK versions.
```
Comment (guybashan) [inline README.md]: **Please Note** that the demo apps listed here **might not contain** the latest updated SDK.

Comment (guybashan) [reply README.md]: pls not the bold parts..

Comment (rachelhiayouappi) [reply README.md]: updated

**Review by guybashan — APPROVED** [review]

**Pull request merged**